### PR TITLE
Uses ReplyBuilder.js to compute the list of recipients when replying to a email.

### PR DESCRIPTION
**Pull request by StCyr**
this PR can also be used to test #2448

Signed-off-by: Cyrille Bollu <cyrpub@bollu.be>

### CODE REVIEW

File: src/components/NewMessageDetail.vue [hunk 1]
```diff
@@ -83,10 +89,13 @@ export default {
 					msgTo = []
 				}
 
+				var account = this.$store.getters.getAccount(message.accountId)
```
Comment (ChristophWurst) [inline src/components/NewMessageDetail.vue]: use `const`. Otherwise looks fine :)

**Comment by ChristophWurst** [comment]
yay, the actions run :)

**Comment by StCyr** [comment]
Here you are :-)

File: src/components/NewMessageDetail.vue [hunk 2]
```diff
@@ -83,10 +89,13 @@ export default {
 					msgTo = []
 				}
 
+				const account = this.$store.getters.getAccount(message.accountId)
```
Comment (ChristophWurst) [inline src/components/NewMessageDetail.vue]: account is not used? :)

**Comment by StCyr** [comment]
looks good now :-)

File: src/ReplyBuilder.js [hunk 3]
```diff
@@ -75,22 +75,22 @@ export const buildRecipients = (envelope, ownAddress) => {
 	let to = []
 	let cc = []
 	if (recipientType === RecipientType.To) {
-		// Send to everyone except yourself plus the original sender
+		// Send to everyone except yourself, plus the original sender if not ourself
 		to = envelope.to.filter(isNotOwnAddress)
-		to = to.concat(envelope.from)
+		to = to.concat(envelope.from.filter(isNotOwnAddress))
```
Comment (ChristophWurst) [inline src/ReplyBuilder.js]: oh that makes sense! Could we add a test case for this? :v:

Comment (StCyr) [reply src/ReplyBuilder.js]: done :)

**Comment by ChristophWurst** [comment]
Tip for a clean commit history and better developer experience: https://dev.to/tmr232/dont-amend-fix :)

**Pull request merged**